public/404.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+     <meta charset="utf-8" />
+     <title>Quizicle</title>
+     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" >
```

Comment:
Also, please remove this if we are not using this. It looks like it was copied from Rob.